pallets/allocations/src/benchmarking.rs

```diff
@@ -52,28 +52,31 @@ fn assert_last_event<T: Config>(generic_event: <T as Config>::Event) {
 }
 
 benchmarks! {
-	batch {
+	allocate {
 		let b in 1..T::MaxAllocs::get();
 
 		let batch_arg = make_batch::<T>(b);
 		let oracle: T::AccountId = account("oracle", 0, SEED);
 		let mut members = <BenchmarkOracles<T>>::get();
 		assert!(members.try_push(oracle.clone()).is_ok());
 		<BenchmarkOracles<T>>::put(&members);
+		<Allocations<T>>::checked_update_session_quota();
 		<SessionQuota<T>>::put(T::ExistentialDeposit::get() * (b * ALLOC_FACTOR).into());
-	}: _(RawOrigin::Signed(oracle), batch_arg)
+	}:{
+		let _ = Allocations::<T>::allocate(batch_arg);
+	}
```

Comment:
We should benchmark for the worst case scenario, looks like line 63 would let the benchmarked extrinsic skip it next time it runs based on my understanding of `checked_update_session_quota`

Reply:
@ETeissonniere Actually the line 63 is not needed any more is a remnant of the previous commits. The function `allocate` doesn't call checked_update_session_quota. I made this separation intentionally to be able to calculate the weights accurately.